README.md

```diff
@@ -91,41 +93,49 @@ statful.Statful{
 Create a simple HTTP API configuration for the client.
 
 ```golang
-statful.Statful{
-    Sender: &statful.BufferedMetricsSender{
-        Client: &statful.ApiClient{
+statful.New(
+    statful.Configuration{
+
+        AutoFlush: false,
+        DryRun: false,
+        FlushInterval: 50,
+        FlushSize: 1000,
+
+        Sender: &statful.HttpSender{
             Http:     &http.Client{},
-            Url:      "https://api.statful.com",
-            BasePath: "/tel/v2.0/",
+            Url:      "https://api.statful.com/tel/v2.0/",
             Token:    "12345678-09ab-cdef-1234-567890abcdef",
         },
-        FlushSize: 1000,
-        Buf:       bytes.Buffer{},
-    },
-    GlobalTags: statful.Tags{"client": "golang"},
-    DryRun:     false,
-}
+
+        Logger: log.New(os.Stderr, "", log.LstdFlags),
+        Tags: statful.Tags{"client": "golang"},
+    }
```

Comment:
I'm pretty sure if this example is not correct and I think that's why you've changed it (if that's the case please update all of the examples on the README for consistency) but I don't understand why we're adding the `AutoFlush` to false here - I suggest we add a new example specifically to initialise an HTTP client with auto flush disabled.

Note: you're also adding the `FlushInterval` here when I think the idea of the example was to simplify and provide an example of how to initialise an HTTP client that flushes based on the buffer size only.
Note 2: the `FlushInterval` config field must be a `time.Duration` and not an int